docs: document c8run zeebe user task issue (#4874)
camunda/camunda-docs: versioned_docs/version-8.6/self-managed/setup/deploy/local/c8run.md

## Shut down Camunda 8 Run

To shut down Camunda 8 Run and end all running processes, run `./shutdown.sh` (or `.\c8run.exe stop` on Windows) from the C8Run directory.

> ## Known Issues

Reviewer: ```suggestion
## Known issues
```

My question regarding this section is around wording alignment regarding "known issues" (same as "limitations"). I wonder if a `note` in the components list would be more effective, particularly as we can use that to encourage any users to move to 8.7, if this issue isn't going to be fixed in 8.6. @theburi is there any product opinion here?

Reviewer 2: I think we can keep it as is. I don't think people will navigate to other pages.

> - Unable to perform operations(assign/unassign/complete) on Camunda User Tasks, formerly known as Zeebe User Tasks, in Tasklist

Reviewer 3: ```suggestion
- In Tasklist, you cannot perform operations (assign/unassign/complete) on Camunda User Tasks (formerly known as Zeebe User Tasks). 
```

> - Unable to perform operations(assign/unassign/complete) on Camunda User Tasks, formerly known as Zeebe User Tasks, in Tasklist

Reviewer: ```suggestion
- In Tasklist, you cannot perform operations (assign/unassign/complete) on Zeebe User Tasks.
```

I believe they're still Zeebe user tasks in 8.6; I am not sure if we plan to backport that name change.
